manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -482,3 +484,5 @@ global:
 
   # Use the Mesh Control Protocol (MCP) for configuring Istiod. Requires an MCP source.
   useMCP: false
+
+istiodRemote: {}
```

Comment:
Why is this needed?

Reply:
Otherwise the helm chart fails since this value is referencing in the chart

Reply:
I was able to install/upgrade without this value so I’m guessing it’s referenced under a conditional?